### ISSUE TITLE
Create user table

### ISSUE THREAD

**Issue by aem** [dbfinal/webapp#2]
- id
- name
- lastLatitude
- lastLongitude
- birthday
- username
- password
- email?
- dateCreated
- dateUpdated

**Comment by aem** [comment]
@nshenkman can you look up the gem for user authentication?

**Comment by nshenkman** [comment]
yup will do. Haven't created the tables yet but will do that after

**Comment by aem** [comment]
i'm working on the other tables right now, so I won't worry about this one until we know how to do auth
